src/layers/XRLayer/XRLayer.js

```diff
@@ -16,7 +16,7 @@ import { padColorsAndSliders } from '../utils';
 const defaultProps = {
   pickable: true,
   coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
-  channelData: { type: 'array', value: {}, async: true },
+  channelData: { type: 'array', value: {}, compare: true },
```

Comment:
Small change leftover from deck.gl upgrade.

Reply:
Just so I know what's the difference here?

Reply:
Can deck.gl just handle async now?

Reply:
How will it compare the channeldata?

Reply:
The `TileLayer` resolves incoming promises now so we don't need the `XRLayer` to be `async` anymore.  This just makes it a standard prop, like the other ones.